sfcasts/assets.md

```diff
@@ -1,96 +1,43 @@
-sidebar BG. So this sidebar BG variable is actually set to another variable, this bar
-syntax. And you'll find that variable in one other file called color palette. That
-SCSS, which is right next to this notice, these are SAS files, but this CSS variable
-system is nothing to do with SAS. It's a pure CSS feature. So you can see lots of
-other variables inside of here. So if you kind of follow the logic here, sidebar BG
-set to gray, 50 and gray 50, let's see, where is the gray inside of here?
-
-You scroll all the way to the bottom. You'll see gray 50, it's the two blue, gray 50.
-And you can kind of go from there where we eventually find where that is. Anyways,
-this is a right way of knowing how to E easy, getting an idea of how to override
-things. What we'll do is I'm actually going to copy this syntax here and the way you
-define CSS variables typically is under this little weird colon root thing. So we're
-gonna do the same thing here. I'm gonna get rid of my body, say colon root, and then
-say cyber BG. And I can, uh, I can reference variables here. That's totally allowed,
-but I'm just gonna replace this with A different hex color. All right. So we try it
-now. Hmm. Did not look like it worked. You can see over here. It's so sad. Let me do
-a command shift. R do force refresh and Got it. So you can see my new D E BFF it's
-subtle, but it is loading the correct color now. So we just customize the assets
-globally for our entire admin section, But we could override configure assets in any
-of our crud controllers. If we needed to do something only for a specific controller.
-
-All right, next, let's start digging into what is quite possibly the most important
-part of configuring easy admin fields. These control, which fields show up on the
-index page, as well as the form pages.
+The EasyAdmin interface looks pretty great out of the box, but what if we want to customize the way something looks? For example, I want to change the background on this sidebar. How can we do that? This type of stuff can be controlled via the `configureAssets()` method. As a reminder, this is a method that exists on both our base controller *and* inside of our individual CRUD controller. So we can control assets on a global level, *or* we can control assets by overriding the same method inside one of our credit controllers. Let's do ours globally so we can change the color of the sidebar for every single page.
 
+Anywhere inside of my `DashboardController`, I'll go back to the "Code Generate" menu, select "OverrideMethods" and override `configureAssets`. This has a lot of cool methods on it. There's some simple ones like `->addCssFile()`. So if you said `->add.CssFile('foo.css')`, that's going to include a link tag to `/foo.css`. As long as I have `/foo.css` inside of my public directory, that would work. The same thing goes for `->addJsFile()`. You can also `->addHtmlContentToBody()` or `->addHtmlContentToHead()`. There's a ton of interesting methods!
```

Comment:
I realized I missed adding the parenthesis at the end of `configureAssets` here. Fixing that in my draft.